### DISCUSSION TITLE
Dedicated Tab for Singleton entities

### DISCUSSION THREAD

**Discussion by pratik149** [directus/directus#5562] [Feature Requests]
It would be really nice if we create a dedicated tab on the left sidebar for Singleton entities, instead of mixing them in the list of Collections itself, because technically speaking Singleton are not Collections. Also it will be more intuitive for the non-technical users!

OR

If creating a tab is too much, then may be we should at least consider showing two different list on the same page by putting a divider between collections and singletons. I have demonstrated the same in the images below.

**Current system**

![collections](https://user-images.githubusercontent.com/31278436/117574656-cdbe1b00-b0fb-11eb-8a9c-a45c9745eb85.PNG)

**Expected system**

![singletons](https://user-images.githubusercontent.com/31278436/117574684-e29aae80-b0fb-11eb-86c1-43219b3c83b2.PNG)

**Comment by Attacler** [comment]
I think this should be done differently.

You can create custom (collection) navigation, that will change a part of the expected system.
But i think we should change this overview to follow the navigation, this way you can change the navigation and the collection overview at the same time.

**Reply by anlexN** [reply]
what is `singleton` meaning? official docs have no explanation.

**Reply by br41nslug** [reply]
within the context of programming a singleton means "a set consisting of one given element."

**Reply by anlexN** [reply]
so can you update [glossary](https://docs.directus.io/getting-started/glossary.html) for `singleton` word?
[strapi](https://strapi.io/) and [ghost](https://ghost.org/) have no `singleton` conception.
I am often confused about some directus words.

**Reply by br41nslug** [reply]
Will add it to the glossary 😄 Can be hard sometimes to decide what words are just plain english and what is programming jargon that needs explaining 😅 You're always welcome to open a PR or discussion if you come across these words that arent explained

**Reply by anlexN** [reply]
1. I and google recommend finding words in [schema.org](https://schema.org/)
2. please check my [pull request](https://github.com/directus/docs/pull/142)
3. thank you very much

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
